src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java

```diff
@@ -62,9 +66,14 @@ public void bindCommands() {
 		RobotMap.Component.driverXbox.dPad.right.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 90,
 			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
 		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
+		RobotMap.Component.teensyStick.getButton(6).whenPressed(new SetOverride(true, new BallioOuttake()));
+		RobotMap.Component.teensyStick.getButton(6).whenReleased(new SetOverride(false, new BallioOuttake()));
```

Comment:
every place that set the hopper(Ballio) now is a runif that only runs when the override is off. There is another branch(the last one I made), which has accounts for all places the hopper is set, shooter included